fix(minor): Changed error message display in printview

marination opened the pull request:
Error will be prompted in the printview of selected print format rather than a pop-up with the file path in it.
![image](https://user-images.githubusercontent.com/25857446/63246223-d6175800-c27f-11e9-8b8b-5bb109026d4c.png)

marination committed:
fix(minor): Changed error message display in printview

netchampfaris commented on frappe/www/printview.py:
```diff
@@ -254,8 +254,14 @@ def get_print_format(doctype, print_format):
 		elif print_format.raw_commands and print_format.raw_printing:
 			return print_format.raw_commands
 		else:
-			frappe.throw(_("No template found at path: {0}").format(path),
```
This is kind of a breaking change. This method throws an error when print_format template is not found but after your change it doesn't. You should handle this at client side. Catch this error and then show the appropriate HTML.

marination replied:
aight

rmehta replied:
update?

marination replied:
Need to clarify with Faris, working on it

marination committed:
fix: Check for HTML on creation of Print Format

The printview will remain as it is.
Validation of HTML while Custom Format is checked and Raw Printing isn't checked in Print Format.

marination committed:
fix: Improved handling of no template found in Printview

User will see 'No preview found' instead of error prompt with path in it
Validation on Print Format refactored slightly.

marination committed:
Codacy fixes
